src/Init/Util.lean

```diff
@@ -30,17 +30,22 @@ def dbgStackTrace {α : Type u} (f : Unit → α) : α := f ()
 @[extern "lean_dbg_sleep"]
 def dbgSleep {α : Type u} (ms : UInt32) (f : Unit → α) : α := f ()
 
-@[noinline] private def mkPanicMessage (modName : String) (line col : Nat) (msg : String) : String :=
-  "PANIC at " ++ modName ++ ":" ++ toString line ++ ":" ++ toString col ++ ": " ++ msg
+structure StackInfo where
+  modAndDecl : String
+  line : Nat
+  col : Nat
```

Comment:
```suggestion
structure CallerInfo where
  module : Name
  declName : Name
  position : Lean.Position
```

This isn't a stack trace (which the name stack info suggests, at least to me).

Having the module/declaration name available directly could make this more generally useful.

Reply:
`Lean.Name` is in the prelude so there is no issue.

I also think it would be better to move the `Lean.Position` structure to `init` if that means we can use the type consistently everywhere.